crates/csharp/src/csproj.rs

```diff
@@ -3,7 +3,9 @@ use std::{fs, path::PathBuf};
 
 use heck::ToUpperCamelCase;
 
-pub struct CSProject {
+pub struct CSProject;
+
+pub struct CSProjectLLVMBuilder {
```

Comment:
Somewhere in here we will need to add the DIrectPInvoke instructions.  The WasmImportLinkageAttribute tells the compiler to emit the wasm import attributes to LLVM, but these methods also need to be "Direct PInvokes" from a NAOT perspective https://learn.microsoft.com/en-us/dotnet/core/deploying/native-aot/interop#direct-pinvoke-calls.  This means adding someting like 
```cs
 <ItemGroup>
   <DirectPInvoke Include="xyz!SimpleDirectPInvokeTestFunc" />
 </ItemGroup>
```
where xyz is the Wasm import module name, and SimpleDirectPInvokeTestFunc is the Wasm import function name
or perhaps better, more concise
```cs
 <ItemGroup>
   <DirectPInvoke Include="xyz />
 </ItemGroup>
```
which will mark all exports in the `xyz` module as direct P/invokes.

Reply:
We don't actually need this now, as Naot for Wasm treats all `WasmImportLinkage` marked methods as direct pInvoke